backup/bin/barman-save-metrics

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# NOTE: not run with '-e' because we need to continue even if the
+# check/status commands fail.
```

Comment:
Why not -e except for those two lines?

Reply:
Doing that makes the script look a bit weird because we have set -e changing 3 times (top, above the `barman check` line, below the `barman status` line) but the lines that can cause set -e to throw are only `mkdir -p` (can't really fail), `rm -f` (can't fail), a test that's designed to exit and the final metrics parsing

Reply:
Actually, the last line should always be run too